src/main/java/be/seeseemelk/mockbukkit/block/state/ContainerMock.java

```diff
@@ -121,4 +124,13 @@ public void setCustomName(@Nullable String name)
 		return ((InventoryMock) this.inventory).getSnapshot();
 	}
 
+	@Override
+	public EnumInteractionResult simulateUse(PlayerMock playerMock, EquipmentSlot equipmentSlot)
+	{
+		if (inventory != null) {
+			playerMock.openInventory(inventory);
+		}
+		return EnumInteractionResult.CONSUME;
```

Comment:
This is NMS logic and it consumes the interaction not the item

Reply:
Oh, I understand. Maybe it would be better if it's called something else though; we don't need to copy nms code, alot of nms code is actually a mess.

What about renaming all of the interaction results to `.*_INTERACTION`, for example `CONSUME_INTERACTION`